app/models/activity_session.rb

```diff
@@ -300,13 +300,7 @@ def update_classroom_activity
   end
 
   def update_milestones
-    # more milestones can be added here as relevant, for now this just checks to see if a Completed Diagnostic milestone needs to be created
-    if self.state == 'finished' && self.classroom_activity_id && self.classroom_activity.activity.activity_classification_id === 4
-      teacher_milestones = self.classroom_activity.unit.user.milestones
-      if !teacher_milestones.find_by(name: 'Complete Diagnostic')
-        teacher_milestones.push(Milestone.find_by(name: 'Complete Diagnostic'))
-      end
-    end
+    UpdateMilestonesWorker.perform_async(self.uid)
```

Comment:
Using UID instead of ID because I noticed this was the pattern elsewhere in this file. Is this correct, and why is it done this way?